### PR TITLE
docs(FEC-12669): Add docs for 'change media' and 'KalturaPlayer.configure()' API

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

 Add docs for 'change media' 

solves: FEC-12669

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: docs/change-media.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Change Media
+
+The Player enables you to play several different media in the same instance of the player,
+All you have to do is call the `setMeida()` or `loadMeida()` API with the new `entryId / source`
```
Comment (giladna) [inline docs/change-media.md:4]: is to call

File: docs/change-media.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      kalturaPlayer.loadMedia({entryId: '{ANOTHER_ENTRY_ID}'});
+    ), 5000);
+
+  } catch (e) {
+    console.error(e.message);
+  }
+</script>
+```
+
+In that case, the player resets, and will play the newly provided media.
```
Comment (giladna) [inline docs/change-media.md:35]: player will call reset internally and ...

File: docs/change-media.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+The reset only includes everything related to the source / source metadata, **but keep the reset of the settings** which are not media specific, such as configuration / plugins
```
Comment (giladna) [inline docs/change-media.md:37]: reset includes only data structures which are related ...
**but keep the reset of the settings as is**

File: docs/change-media.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+If you want to also change any settings or plugins configuration between different media, you can do this by using `configure` API
```
Comment (giladna) [inline docs/change-media.md:39]: if you want to change

File: docs/player-setup.md [hunk 5]
```diff
@@ -88,7 +88,7 @@ var config = {
 };
 ```
 
-> For full configuration details see [this]() document.
+> For full configuration details see [this](./configuration.md) document.
```
Comment (giladna) [inline docs/player-setup.md:91]: need to add to the guide the link to change media

File: docs/change-media.md (hunk 1 above)
Comment (JonathanTGold) [reply docs/change-media.md:4]: fixed

File: docs/change-media.md (hunk 2 above)
Comment (JonathanTGold) [reply docs/change-media.md:35]: fixed

File: docs/change-media.md (hunk 4 above)
Comment (JonathanTGold) [reply docs/change-media.md:39]: fixed

**Pull request merged**